<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0,0">
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Pomodoro</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0,0">
          <item row="0" column="0">
           <widget class="QCheckBox" name="pomodoroTimer">
            <property name="text">
             <string>Pomodoro timer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="pomodoroMinutes">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>25</string>
            </property>
            <property name="maxLength">
             <number>3</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="pomodoroMinutesLabel">
            <property name="text">
             <string>minutes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="pomodoroBreakTimer">
            <property name="text">
             <string>Pomodoro break timer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pomodoroBreakMinutes">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>25</string>
            </property>
            <property name="maxLength">
             <number>3</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="pomodoroBreakMinutesLabel">
            <property name="text">
             <string>minutes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="idleMinutes">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="idleMinutesLabel">
         <property name="text">
          <string>minutes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="idleDetection">
         <property name="text">
          <string>Idle detection</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="recordTimeline">
         <property name="text">
          <string>Record timeline</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="shortcutsGroupBox">
         <property name="title">
          <string>Shortcuts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="1,0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="continueStopLabel">
            <property name="text">
             <string>Continue/Stop timer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="continueStopButton">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Record Shortcut</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="continueStopClear">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Clear shortcut</string>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="showHideLabel">
            <property name="text">
             <string>Show/Hide Toggl</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="showHideButton">
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Record Shortcut</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="showHideClear">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Clear Shortcut</string>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="focusAppOnShortcut">
            <property name="text">
             <string>Focus app on shortcut</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="stopEntry">
         <property name="text">
          <string>Stop running entry on computer sleep/shutdown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_proxy">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="useSystemProxySettings">
         <property name="text">
          <string>Use system proxy settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="useProxy">
         <property name="title">
          <string>Use prox&amp;y to connect to Toggl</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="hostLabel">
            <property name="text">
             <string>&amp;Host</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="proxyHost"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="portLabel">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="proxyPort"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="useranameLabel">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="proxyUsername"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="password_label">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="proxyPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_reminder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Reminder</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="3,0,0">
       <item row="3" column="1" colspan="2">
        <widget class="QTimeEdit" name="reminderEndTimeEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="remindToTrackTime">
         <property name="text">
          <string>Remind to track time</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_7">
         <property name="text">
          <string>Sunday</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="reminderEndTimeLabel">
         <property name="text">
          <string>Reminder end time</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_4">
         <property name="text">
          <string>Thursday</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="reminderStartTimeLabel">
         <property name="text">
          <string>Reminder start time</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_3">
         <property name="text">
          <string>Wednesday</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="reminderDaysLabel">
         <property name="text">
          <string>Reminder days</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_5">
         <property name="text">
          <string>Friday</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_6">
         <property name="text">
          <string>Saturday</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_1">
         <property name="text">
          <string>Monday</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="reminderMinutes">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QCheckBox" name="dayCheckbox_2">
         <property name="text">
          <string>Tuesday</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QTimeEdit" name="reminderStartTimeEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="reminderMinutesLabel">
         <property name="text">
          <string>minutes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
